tracetesting/run.bash

```diff
@@ -61,7 +59,7 @@ run_test_suite_for_feature() {
   junit_output='results/'$feature'_test_suite.xml'
   definition='./features/'$feature'/_test_suite.yml'
 
-  $TRACETEST_CLI_MAIN --config ./config.main.yml test run --definition $definition --environment ./.main.env --wait-for-result --junit $junit_output
+  $TRACETEST_CLI --config config.yml test run --definition $definition --environment .env --wait-for-result --junit $junit_output
```

Comment:
Since we don't have to connect on `integration` and `target` environment at same time anymore, I've removed the references for `tracetest-main` for now.
Later we can add a README here to explain better how this dogfooding test is working.